runtime/queries/snakemake/highlights.scm

```diff
@@ -18,7 +18,7 @@
     (_
       (directive
         name:
-          _ @label)))
+          _ @property)))
```

Comment:
This looks bad :thinking: 
I think I f'd up something in the formatter. This kind of pattern shouldn't happen

Will fix later

Reply:
#5885